docs/FAQ.md

```diff
@@ -109,10 +109,41 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I deal with divergent changes ('??' after the ID)?
+
+A [divergent change][glossary_divergent_change] represents a change that has two
+or more visible commits associated with it. To refer to such commits, you must
```

Comment:
I still feel that the word "change id" should be somewhere in here, e.g. "... associated with a single change id".

Reply:
If this was already discussed and rejected, it doesn't have to be blocking.

Reply:
Ah, sorry, missed that. Let me add it somewhere.